packages/editor-ui/src/components/MainSidebar.vue

```diff
@@ -339,6 +346,14 @@ export default mixins(
 		}
 		await Vue.nextTick();
 		this.fullyExpanded = !this.isCollapsed;
+
+		try {
+			const planData = await this.usersStore.getOwnerCurrentPLan();
```

Comment:
should we be calling this here? could we do this when after the app is mounted and user is logged in (App.vue) instead? that seems like a more apt place especially since we agreed to add some polling here for trial users as well.. every time the endpoint is pulled, we can update the store with these values and vue ractivity can update the components